pyproject.toml

```diff
@@ -156,17 +159,21 @@ extend-exclude = [
 [tool.ruff.flake8-tidy-imports]
 ban-relative-imports = "all"
 
-
-[tool.isort]
-py_version = 37
-profile = "black"
-force_single_line = true
-combine_as_imports = true
-lines_between_types = 1
-lines_after_imports = 2
-src_paths = ["src", "tests"]
-extend_skip = ["setup.py"]
-known_third_party = ["poetry.core"]
+[tool.ruff.isort]
+force-single-line = true
+combine-as-imports = true
+lines-between-types = 1
+lines-after-imports = 2
+known-first-party = ["poetry"]
+known-third-party = ["poetry.core"]
+required-imports = ["from __future__ import annotations"]
+section-order = [
+    "future",
+    "standard-library",
+    "third-party",
+    "first-party",
+    "local-folder"
+]
```

Comment:
```suggestion
```

This seems to be the default order, so we don't have to configure it.

Reply:
According to the [documentation](https://beta.ruff.rs/docs/settings/#section-order) by default this value is empty.

Reply:
The [documentation of isort](https://pycqa.github.io/isort/docs/configuration/custom_sections_and_ordering.html) seems to be better in this point:

> You can change the section order with sections option from the default of:
> 
> `FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER`

I'm not good at Rust but after some trial and error and code spelunking, I'd say ruff does the same:
[ImportType Enum](https://github.com/charliermarsh/ruff/blob/64b7280eb824d0e5f9da887e82dcda53838dd38d/crates/ruff/src/rules/isort/categorize.rs#L35C1-L41)
[Default order 1](https://github.com/charliermarsh/ruff/blob/64b7280eb824d0e5f9da887e82dcda53838dd38d/crates/ruff/src/rules/isort/settings.rs#LL343C83-L343C83)
[Default order 2](https://github.com/charliermarsh/ruff/blob/64b7280eb824d0e5f9da887e82dcda53838dd38d/crates/ruff/src/rules/isort/settings.rs#L351-L353)